spec/models/user_rate_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'spec_helper'
+
+describe "User Rate" do
+  it "has a valid factory" do
```

Comment:
you can use `FactoryGirl.lint` method to verify that your factories are correct without need to add this example to each model spec :)

http://www.rubydoc.info/gems/factory_girl/file/GETTING_STARTED.md#Linting_Factories